vue-app/src/components/GridConfigSlider.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <label>{{ _name }}</label>
+  <input
+    type="range"
+    min="1"
+    max="300"
+    name="{{_name}}"
+    :value="_value"
+    @input="emit('GSchange', $event)"
+  />
+</template>
+
+<script setup lang="ts">
+defineProps<{
+  _value: string | number | undefined;
```

Comment:
maybe change the name of prop to use usual variable naming standard, without "_"